comp/core/bundle.go

```diff
@@ -34,8 +34,8 @@ var Bundle = fxutil.Bundle(
 
 // MockBundle defines the mock fx options for this bundle.
 var MockBundle = fxutil.Bundle(
-	fx.Provide(func(params BundleParams) config.Params { return params.ConfigParams }),
-	config.Module,
-	fx.Provide(func(params BundleParams) log.Params { return params.LogParams }),
-	log.Module,
+	fx.Supply(config.Params{}),
```

Comment:
Was this change done on purpose?

Reply:
Yes - in order to run tests with the core bundle, it makes sense to use the mocks otherwise you will have to provide a valid config (on disk) and other startup settings needed to run a "real" agent. So I thought it made sense to update the core `MockBundle` to use the related mocks. [You can see how this is used here](https://github.com/DataDog/datadog-agent/pull/15283/files#diff-5b298cbbe1b2f9a80ffeb38c27a48d3789f6439105a75c17c1a8a439ab133451R34)